snapraid-runner.py

```diff
@@ -45,11 +45,13 @@ def tee_thread():
     return t
 
 
-def snapraid_command(command, args={}, *, allow_statuscodes=[]):
+def snapraid_command(command, args={}, *, allow_statuscodes=[], log_output = False):
```

Comment:
IMO `log_ouput` is not the right name for this. The output is always logged. It's just that if the user sets the email preference to `short`, stdout is not included in the email. `increase_output_loglevel` would be better. Also please follow pep8.

```suggestion
def snapraid_command(command, args={}, *, allow_statuscodes=[],
                     increase_output_loglevel=False):
```